src/inmanta/agent/agent.py

```diff
@@ -1331,6 +1331,17 @@ async def trigger_update(self, env: uuid.UUID, agent: str, incremental_deploy: b
         )
         return 200
 
+    @protocol.handle(methods.trigger_release_version, env="tid", agent="id")
+    async def release_version(self, env: uuid.UUID, agent: str, incremental_deploy: bool) -> Apireturn:
+        """
+        Trigger an update
+        """
+        return await self.trigger_update(
+            env=env,
+            agent=agent,
+            incremental_deploy=incremental_deploy,
+        )
```

Comment:
This should be empty I think?

Reply:
Why? We want to preserve the old behaviour,. The `trigger_update` method was called before this refactor